package.json

```diff
@@ -4,13 +4,14 @@
   "version": "0.0.0",
   "main": "dist/index.html",
   "scripts": {
-    "start": "./node_modules/.bin/gulp",
-    "build": "./node_modules/.bin/gulp build",
-    "test": "./node_modules/.bin/gulp test",
-    "serve": "./node_modules/.bin/gulp serve",
-    "deploy": "./node_modules/.bin/gulp deploy"
+    "start": "gulp",
+    "build": "gulp build",
+    "test": "gulp test",
+    "serve": "gulp serve",
+    "deploy": "gulp deploy"
   },
   "dependencies": {
+    "gulp-cli": "github:gulpjs/gulp-cli#4.0",
```

Comment:
We had this discussion before.
We want a clear distinction between what you need to run the website and what you need only to build it. Especially now that we replaced bower with npm.
I think this is a common standard looking at the latest boilerplates out there.
Can we agree on that close this discussion?